docs/tutorial-video/YodaSpeak/Integrations/YodaSpeak/YodaSpeak.py

```diff
@@ -0,0 +1,103 @@
+# uncomment the import statements for debugging in PyCharm, VS Code or other IDEs.
+# import demistomock as demisto
+# from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+# from CommonServerUserPython import *  # noqa
+
+TRANSLATE_OUTPUT_PREFIX = 'TheForce'
```

Comment:
I must admit that `TheForce` still makes no sense. It's a nice joke, but I think it'll be clearer if we use a more meaningful object name, such as `Phrase` or `Sentence`. i.e. the `original` phrase is X and its `translation` is Y.